WindowsAzure/WindowsAzure.php

```diff
@@ -1,335 +1,376 @@
-            'windowsazure\\table\\models\\filters\\filter' => '/Table/Models/Filters/Filter.php',
-            'windowsazure\\table\\models\\filters\\propertynamefilter' => '/Table/Models/Filters/PropertyNameFilter.php',
-            'windowsazure\\table\\models\\filters\\querystringfilter' => '/Table/Models/Filters/QueryStringFilter.php',
-            'windowsazure\\table\\models\\filters\\unaryfilter' => '/Table/Models/Filters/UnaryFilter.php',
-            'windowsazure\\table\\models\\getentityresult' => '/Table/Models/GetEntityResult.php',
-            'windowsazure\\table\\models\\gettableresult' => '/Table/Models/GetTableResult.php',
-            'windowsazure\\table\\models\\insertentityresult' => '/Table/Models/InsertEntityResult.php',
-            'windowsazure\\table\\models\\property' => '/Table/Models/Property.php',
-            'windowsazure\\table\\models\\query' => '/Table/Models/Query.php',
-            'windowsazure\\table\\models\\queryentitiesoptions' => '/Table/Models/QueryEntitiesOptions.php',
-            'windowsazure\\table\\models\\queryentitiesresult' => '/Table/Models/QueryEntitiesResult.php',
-            'windowsazure\\table\\models\\querytablesoptions' => '/Table/Models/QueryTablesOptions.php',
-            'windowsazure\\table\\models\\querytablesresult' => '/Table/Models/QueryTablesResult.php',
-            'windowsazure\\table\\models\\tableserviceoptions' => '/Table/Models/TableServiceOptions.php',
-            'windowsazure\\table\\models\\updateentityresult' => '/Table/Models/UpdateEntityResult.php',
-            'windowsazure\\table\\tablerestproxy' => '/Table/TableRestProxy.php'
-          );
-      }
-      $cn = strtolower($class);
-      if (isset($classes[$cn])) {
-         require __DIR__ . $classes[$cn];
-      }
-   }
-);
+<?php
+
+/**
+ * LICENSE: Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * PHP version 5
+ *
+ * @category   Microsoft
+ * @package    WindowsAzure
+ * @author     Azure PHP SDK <azurephpsdk@microsoft.com>
+ * @copyright  2012 Microsoft Corporation
+ * @license    http://www.apache.org/licenses/LICENSE-2.0  Apache License 2.0
+ * @link       https://github.com/windowsazure/azure-sdk-for-php
+ * @deprecated deprecated since version 0.4.2 and replaced by vendor/autoload.php
+ */
+
+trigger_error(sprintf('Usage of `%s` has been deprecated since version 0.4.2 and will be removed in later versions. ' .

```

Comment:
@yaqiyang, any plan to respect semantic versioning? "later versions" sounds too ambiguous.